WordPress/Classes/ViewRelated/Reader/FollowedSitesViewController.m

```diff
@@ -204,19 +204,16 @@ - (NSString *)tableView:(UITableView *)tableView titleForDeleteConfirmationButto
     return NSLocalizedString(@"Unfollow", @"Label of the table view cell's delete button, when unfollowing a site.");
 }
 
-- (NSString *)titleForHeaderInSection:(NSInteger)section
+- (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section
 {
     if ([[self.tableViewHandler.resultsController fetchedObjects] count] > 0) {
         return NSLocalizedString(@"Sites", @"Section title for sites the user has followed.");
     }
-    // Return an space instead of empty string or nil to preserve the section
-    // header's height if all items are removed and then one added back.
-    return @" ";
+    return nil;
 }
 
 - (void)tableViewDidChangeContent:(UITableView *)tableView
 {
-    [self.tableViewHandler updateTitleForSection:0];
```

Comment:
Yes it does 😅, this can be tested by loading the Reader "Followed Sites" for the first time (empty table) and then upon load the tableView will display the section title.